src/ape/api/accounts.py

```diff
@@ -208,6 +210,23 @@ def deploy(self, contract: "ContractContainer", *args, **kwargs) -> "ContractIns
             _contract_type=contract.contract_type,
         )
 
+    def check_signature(
+        self,
+        data: Union[SignableMessage, TransactionAPI],
+        signature: Optional[_Signature] = None,  # TransactionAPI doesn't need it
+    ) -> bool:
+        if isinstance(data, SignableMessage):
+            if signature:
+                return self.address == Account.recover_message(data, vrs=signature)
```

Comment:
We don't need to call `encode_vrs()` because the `vrs=` kwarg on `recover_message()` expects a tuple.
Whereas if we use the `signature=` kwarg on `recover_message()`, we have to call `encode_rsv()` because it expects a concatenated string.